### ISSUE TITLE
Some sections don't work - "Artifacts" "Formulas & Scrolls"

### ISSUE THREAD

**Issue by Sir-Mac** [fonsleenaars/tqdb#59]
Hi, on the front end of tq-db.net, the sections "Artifacts" and "Formulas & Scrolls" are just plain text, there is no content behind them, because they are not URL's.

I kindly ask you to fix this, because I would like to preview the new scrolls, artifacts and recipes (formulas) from the second addendum "Atlantis".

Thank you in advance,
Sir-Mac

**Comment by fonsleenaars** [comment]
Are you on the english version?

I see the pages just fine, but if you're on another language, perhaps I missed something.

**Comment by Sir-Mac** [comment]
Hi again, 

switching to another language doesn't change anything...  I made a screenshot, so you can see, what am I exactly talking about.

![tq-db-trouble](https://user-images.githubusercontent.com/50591247/57640757-915e8f80-75b3-11e9-9f79-7344dfeb28e5.png)
As you can see, on the left bottom navigation panel, the sections "Artifacts" "Formulas & Scrolls" are just plain text, no URLS, and the text even don't have the yellow stripe, which will cover the URL, if you move your mouse cursor over it.

**Comment by fonsleenaars** [comment]
Ah, I see the bug. There's a lot more underneath there, but the menu on the left doesn't scroll it looks like (that's my mistake). I'll push a fix for it when I have some time.

This is the full menu:
![image](https://user-images.githubusercontent.com/4301261/57642305-fc44a200-7584-11e9-87ce-31e89bd7cf8c.png)

**Comment by Sir-Mac** [comment]
Thank you for fixing the issue, Mr. Leenaars, I'll wait for that. 

Don't You know, by any chance, who translates the texts of the game from english to other languages? If it is done by manpower, or if it is automatically translated? Because czech localization of the game has numerous issues. The sentences and other semantic structure is perfect, but the Greek, Egyptian, Asian and Nordic names, places and specific terms are translated incorrectly, or are not translated at all.

I did for myself some partial fixes (I unpacked the TEXT_CZ.ARC and did some corrections and many corrections, I will have been doing, when I have proper books with czech mythical vocabulary.)

Thank you once again, Sir-Mac
